qemu/tests/virtio_console.py

```diff
@@ -280,7 +280,7 @@ def test_rw_host_offline_big_data():
         guest_worker.cmd("virt.clean_port('%s'),1024" % port.name, 10)
         match, tmp = guest_worker._cmd("virt.send('%s', (1024**3)*3, True, "
                                        "is_static=True)" % port.name, 30)
-        if match is None:
+        if match is not None:
```

Comment:
Can you please share what is the error you are seeing with `if match is None`? 
And how are you doing handle the case of when `match is None`
Thanks!!